src/Explorer/Panes/AddCollectionPane.html

```diff
@@ -287,62 +241,6 @@
               </div>
               <!-- Unlimited option button - End -->
           </div>
-
-          <!-- Fixed Button Content - Start -->
-          <div class="tabcontent" data-bind="visible: isFixedStorageSelected() && !databaseHasSharedOffer()">
```

Comment:
This may look like it is used, but it is not. I verified on prod this code UI does not ever show. The only place we still do fixed is Mongo